.editorconfig

```diff
@@ -2,11 +2,8 @@ root = true
 
 [*]
 charset = utf-8
-indent_style = tab
+indent_style = space
```

Comment:
Code should always be indented with tabs, not spaces, and the `indent_size` should not be set. This allows everyone to use the indentation level they prefer.

The only exception should be .json files, specifically `package.json` since it gets rewritten by npm/yarn using 2 spaces for indentation.

Reply:
That would lead to inconsistent formatting if people can choose indentation.

I chose to use spaces, since that's more commonly used nowadays.

Reply:
The formatting would be tab-based for everyone. People can configure their IDE to display a `tab` character as any width, but in the file that indentation will always be represented by a `tab` character. This makes the code more accessible to a wider range of developers with different requirements.